src/cdk/nag-suppression.ts

```diff
@@ -32,17 +33,28 @@ export const nagSuppression = async (params: RuleParamsNagSuppression) => {
     Promise.all.bind(Promise),
 
     R.forEach(async (path: string) => {
-      const diff = await danger.git.diffForFile(path)
+      const diff = await danger.git.structuredDiffForFile(path)
 
       if (!diff) {
         return
       }
 
-      const matches = diff.added.match(/\.addResourceSuppressions\(/g)
+      const resourceSuppressionsRegex = /\.addResourceSuppressions\(/
 
-      if (matches) {
-        warn(`Found ${matches.length} new NAG suppression calls`, path)
-      }
+      R.compose<
+        [chunks: Chunk[]],
+        Array<Array<Change>>,
+        Array<Change>,
+        Array<AddChange>,
+        Array<AddChange>,
+        Array<AddChange>
+      >(
+        R.forEach(({ln}) => warn(`Found new NAG suppression call in file "${path}" at line ${ln}`, path, ln)),
```

Comment:
@quesabe does it post a comment on particular line in the code?

Reply:
I can't say for sure since I do not have a repo with setup danger CI to test it... But it should - all the functions provided by danger has the third argument `line` that is described as follows: `@param line — the line which this message should be attached to`.